app/scripts/HiGlassComponent.js

```diff
@@ -2856,6 +2856,36 @@ class HiGlassComponent extends React.Component {
     }
   }
 
+  validateLocks(locks) {
+    // locks are organized like this:
+    // { v1: { v1: [0,0,0], v2: [1,1,1]}}
+    // v1 marks the name of a lock group and also
+    // corresponds to a view uid (a view cannot belong)
+    // to more than one lock group. The numbers are the
+    // zoom, xOffset, yOffset values that were present when
+    // the view was created
+
+    const toRemove = [];
+
+    for (const viewUid of dictKeys(locks)) {
+      if (!locks[viewUid] || !locks[viewUid][viewUid]) {
+        // we need to have the starting location of the
+        // lock's namesake view
+        toRemove.push(viewUid);
+      }
+    }
+
+    for (const viewUid of toRemove) {
+      if (locks[viewUid]) {
+        console.warn(
+          `View ${viewUid} not present in lockGroup: ${locks}. Removing...`
```

Comment:
I am seeing the console warning 
```
View aa not present in lockGroup: [object Object]. Removing...
```
Should the locks object be stringified?

Reply:
Yeah... or removed. Knowing the view id that's absent should be enough to diagnose.

Reply:
I removed the object from the print statement.